SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -57,27 +48,25 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
 
 * **Configuration**
 
-    * **Project Root Folder**</br>
-    If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<ProjectFileLocation>/../spatial`.
+    * **Project root folder**</br>
+    If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`.
 
-    * **Launch Configuration**</br>
+    * **Launch configuration**</br>
     The [launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config) (SpatialOS documentation) to use when running `spatial local launch` using the `Launch` button.
 
-    * **Stop on Exit**</br>
+    * **Stop on exit**</br>
     If enabled, shuts down running deployments when you close the Unreal Editor.
 
     * **Snapshot path**</br>
-    Use this to specify the filepath of your Unreal GDK snapshot. If you leave this empty, it defaults to `<ProjectFileLocation>/../spatial/snapshots`.
+    Use this to specify the filepath of your snapshot. If you leave this empty, it defaults to `<GameRoot>/../spatial/snapshots`.
 
     * **Snapshot file name**</br>
-    Name of your SpatialOS Unreal GDK snapshot file.
+    Name of your snapshot file.
 
-* **Schema Generator**
+* **Schema Generation**
 
-    * **Generate Schema for all Supported Classes**</br>
-    **Experimental** Use this to generate schema for all `AActor` and `UActorComponent` classes (which *don't* extend `USceneComponent`) without explicitly tagging them as `SpatialType` within their UCLASS defintion. 
-    `USceneComponent` is unsupported due to there often being multiple of these components on an Actor which isn't currently supported.
-    If you have `UActorComponent`s which don't extend `USceneComponent` but exist in multiple on an Actor, *this can potentially crash your game*. Make sure there are never multiple components of the same type on an Actor. If there are make sure you modify your Actor to only have one, or use the explicit `SpatialType` tagging approach and don't tag the duplicate `UActorComponent`.
+    * **Generate schema for all supported classes**</br>
+    **Experimental** Use this to generate schema for all UObjects that have replicated or handover properties. As the GDK does not currently support multiple replicated Actor components of the same type on an Actor, classes extended from `USceneComponent` doesn't generate schemas.
```

Comment:
Link to schema in the glossary ({{URLRoot}}/content/glossary#schema)
Also: 
"...classes extended from `USceneComponent` doesn't generate schemas." 
-> grammar!
""..classes extended from `USceneComponent` don't generate schemas.""

Reply:
Done (in GDoc)